Flat/src/ModulesFatJets.cc

```diff
@@ -758,12 +772,12 @@ void PandaAnalyzer::FatjetMatching()
         }
 
       } // loop over subjets
-
-      EvalBTagSF(sj_btagcands,sj_sf_cent,GeneralTree::bCent,GeneralTree::bSubJet);
-      EvalBTagSF(sj_btagcands,sj_sf_bUp,GeneralTree::bBUp,GeneralTree::bSubJet);
-      EvalBTagSF(sj_btagcands,sj_sf_bDown,GeneralTree::bBDown,GeneralTree::bSubJet);
-      EvalBTagSF(sj_btagcands,sj_sf_mUp,GeneralTree::bMUp,GeneralTree::bSubJet);
-      EvalBTagSF(sj_btagcands,sj_sf_mDown,GeneralTree::bMDown,GeneralTree::bSubJet);
+      bool do2=true;
```

Comment:
What's this?

Reply:
it's local and used on the very next line, trying to keep the code self explanatory

Reply:
It's not used on the next line.

Reply:
Actually I'd prefer we just pass literals. The other 4 arguments of the function aren't exactly self-explanatory.

Reply:
ok